auto-run-nondex/valid_repo_extractor.py

```diff
@@ -25,11 +28,17 @@ def extract_org_repos_with_pom_file():
     repos = soup.find_all('a', class_='d-inline-block', href=True)
     while len(repos) > 3:
         for repo in repos:
-            if repo.has_attr('data-hovercard-type') and repo.get('data-hovercard-type') == 'repository':
```

Comment:
Why not do just `get` instead of `has_attr` and `get`?

Reply:
I removed the redundant check